recipes/tinyspline/all/test_package/test_package.cpp

```diff
@@ -1,4 +1,8 @@
-#include <tinysplinecpp.h>
+#ifdef TINYSPLICE_API_0_3
```

Comment:
TINYSPLICE -> TINYSPLINE

Reply:
```suggestion
#ifdef TINYSPLINE_API_0_3
```

Reply:
Thanks for double checking.
I've made this typo before :smiley: